src/utils/mcodeUtils10.js

```diff
@@ -0,0 +1,204 @@
+    '384692006', // Intracavitary brachytherapy (procedure) (B)
+    '113120007', // Interstitial brachytherapy (procedure) (B)
+    '33195004', // Teleradiotherapy procedure (procedure) (B)
+    '385798007', // Radiation therapy care (regime/therapy) (B)
+    '108290001', // Radiation oncology AND/OR radiotherapy (procedure) (B)
+  ],
+
+  tumorMarkerTestCodes: [
+    '85319-2', // HER2 [Presence] in Breast cancer specimen by Immune stain (B)
+    '85318-4', // HER2 [Presence] in Breast cancer specimen by FISH (B)
+    '85337-4', // Estrogen receptor Ag [Presence] in Breast cancer specimen by Immune stain (B)
+    '85339-0', // Progesterone receptor Ag [Presence] in Breast cancer specimen by Immune stain (B)
+    '2857-1', // Prostate specific Ag [Mass/volume] in Serum or Plasma (P)
+  ],
+
+  medicationCodes: [
+    '1114085', // 100 ML zoledronic acid 0.04 MG/ML Injection (B)
+    '993452', // 1 ML denosumab 60 MG/ML Prefilled Syringe (B)
+    '1737449', // 10 ML Pamidronate Disodium 3 MG/ML Injection (B)
+    '1732186', // 100 ML Epirubicin Hydrochloride 2 MG/ML Injection (B)
+    '1790099', // 10 ML Doxorubicin Hydrochloride 2 MG/ML Injection (B)
+    '583214', // Paclitaxel 100 MG Injection (B)
+    '727762', // 5 ML fulvestrant 50 MG/ML Prefilled Syringe (B)
+    '200064', // letrozole 2.5 MG Oral Tablet (B)
+    '199224', // anastrozole 1 MG Oral Tablet (B)
+    '310261', // exemestane 25 MG Oral Tablet (B)
+    '1658084', // ado-trastuzumab emtansine 100 MG Injection (B)
+    '672149', // lapatinib 250 MG Oral Tablet (B)
+    '1940648', // neratinib 40 MG Oral Tablet (B)
+    '2119714', // 5 ML hyaluronidase-oysk 2000 UNT/ML / trastuzumab 120 MG/ML Injection (B)
+    '1946840', // Verzenio 100 MG Oral Tablet (B)
+    '1601380', // palbociclib 100 MG Oral Capsule (B)
+    '1873983', // ribociclib 200 MG Oral Tablet (B)
+    '198240', // Tamoxifen 10 MG Oral Tablet (B)
+    '1734919', // Cyclophosphamide 1000 MG Injection (B)
+    '597195', // Carboplatin 10 MG/ML Injectable Solution (B)
+    '1791701', // 10 ML Fluorouracil 50 MG/ML Injection (B)
+    '1790099', // 10 ML Doxorubicin Hydrochloride 2 MG/ML Injection (B)
+    '1732186', // 100 ML Epirubicin Hydrochloride 2 MG/ML Injection (B)
+    '583214', // Paclitaxel 100 MG Injection (B)
+  ]
+};
+
+const PRIMARY_CANCER_CONDITION_FILTER = 'Condition.code.coding.where($this.code in %primaryCancerConditionCodes)';
+const PRIMARY_CANCER_CONDITION_PATH = fhirpath.compile(PRIMARY_CANCER_CONDITION_FILTER, fhirPathVariables);
+
+const COMORBID_CONDITION_FILTER = 'Condition.code.coding.where($this.code in %comorbidConditionCodes)';
+const COMORBID_CONDITION_PATH = fhirpath.compile(COMORBID_CONDITION_FILTER, fhirPathVariables);
+
+// specific instance of the "find" function with a precompiled fhirpath, possibly a premature optimization but things are already slow on IE
+const findPrimaryCancerCondition = (context) => {
+  if (typeof context === 'object' && context.entry) {
```

Comment:
Unfortunately in JS, this is how `typeof` treats arrays and objects: 
![image](https://user-images.githubusercontent.com/16297930/78799971-dee0a980-7988-11ea-82d7-8f92f8fdf3c2.png)

We have lodash as a dependency, so you could use `_.isObject` instead

Reply:
I don't think it will actually break if it brings in an array, because `context.entry` will be undefined and it won't enter the block. But I'll change that so it does things the right way